src/components/token/TokenConfig.js

```diff
@@ -118,7 +118,7 @@ const TokenConfig = props => {
           <span className="actions-token-code">
             <button className="actions-token-code-btn">
               <CopyToClipboard text={configurationString} onCopy={copied}>
-                <CopyIcon />
+                <CopyIcon style={{ color: 'white' }} />
```

Comment:
Not really, in this specific case, the icon remains white in both themes. It has the inline style because the svg is currentColor (because in other uses of it it changes color).